sse_test.go

```diff
@@ -63,6 +63,7 @@ func TestServer(t *testing.T) {
 			id := fmt.Sprintf("C-%d", n+1)
 			fmt.Printf("Client %s registed to channel %s\n", id, name)
 
+			name := name
```

Comment:
What are you trying to fix here? This was done in a purpose, the exact one that you pointed is the doc, it's not wrong.

Reply:
![image](https://user-images.githubusercontent.com/55949036/101340516-29038d00-3880-11eb-92bb-a2206bf09d6a.png)